webservices/common/models/costs.py

```diff
@@ -69,9 +74,5 @@ def pdf_url(self):
         return utils.make_report_pdf_url(self.beginning_image_number)
```

Comment:
If it is beginning image number, I think that will take the user to the first page of the filing and not the page with the transaction. I think that is different than what we do with other transactions.